packages/frontend/src/helpers/redirect.tsx

```diff
@@ -0,0 +1,12 @@
+import { Navigate } from 'react-router-dom'
+
+import * as URLS from '../config/urls'
+
+export const redirectToLogin = (): JSX.Element => {
```

Comment:
This returns a `Navigate` component, which means it shouldn't be a helper function (e.g. you can't use this in `useEffect`, etc).

Can move this to components/RedirectToLogin or similar.